### DISCUSSION TITLE
when does SuperSLicer pull from upstream/PrusaSlicer

### DISCUSSION THREAD

**Discussion by m10d** [supermerill/SuperSlicer#4494] [General]
I see in the release notes a number of references to pulling from PS (and others like Orca).  

Is superslicer regularly, or ~ever, merging or rebasing up to more recent prusaslicer?  Or are only targeted MR's brought in as need is seen?

**Comment by DoubleStrike** [comment]
He is merging it up to PS 2.7 at the moment.  Once that's done, going to 2.8 and 2.9 will be much simpler.  Since he is starting from 2.5, going to 2.7 incorporates a lot of updates, so it has taken a couple of years and is actively being done now, though he does not provide progress updates too often.
